check_jobs.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Check jobs acceptance timeout.
+# If jobs cannot be confirmed, listening will be terminated.
+
+if [ -z "$1" ]; then exit 1; fi
+
+_RUNNER_WORKDIR=${RUNNER_WORKDIR:-/_work}
+_JOBS_ACCEPTANCE_TIMEOUT=$1
+
+while [ 1 ]; do
+  if [ ! -e ${_RUNNER_WORKDIR}/_temp/_github_workflow ]; then
```

Comment:
i dont follow what this is doing

Reply:
Changed "1" to process check.
Changed check directory, _github_workflow to log directory.